Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/RdbmsProviderTest.cs

```diff
@@ -70,11 +70,55 @@ public override void SetUp ()
 
       _provider = new TestableRdbmsProvider(
           TestDomainStorageProviderDefinition,
+          TestDomainStorageProviderDefinition.ConnectionString,
           NullPersistenceExtension.Instance,
           _commandFactoryMock.Object,
           () => _connectionCreatorMock.Object.CreateConnection());
     }
 
+    [Test]
+    public void RdbmsProvider_WithConnectionStringContainedInProviderDefinition ()
+    {
+      var rdbmsProvider = new RdbmsProvider(
+          new RdbmsProviderDefinition("dummyProviderDefinition", Mock.Of<IRdbmsStorageObjectFactory>(), "dummyConnectionString", "dummyReadOnlyConnectionString"),
+          "dummyConnectionString",
+          NullPersistenceExtension.Instance,
+          _commandFactoryMock.Object,
+          () => _connectionCreatorMock.Object.CreateConnection());
+
+      Assert.That(rdbmsProvider, Is.InstanceOf<RdbmsProvider>());
```

Comment:
That assertion isn't helpful. We are initializing an RdbmsProvider via the ctor and then asserting that the instance is of that type.

If you intend to test that no exception is thrown, you can use `Assert.That(() => new RdbmsProvider(...), Throws.Nothing)`. And since this is the only test dealing with the connection string happy path, I recommend instead asserting that `rdbmsProvider.ConnectionString` is properly assigned.